reporter/src/funTest/assets/static-html-reporter-test-input.yml

```diff
@@ -276,6 +276,11 @@ scanner:
               - path: "fake_path"
                 start_line: 206
                 end_line: 206
+          - license: "GPL-2.0-only WITH Classpath-exception-2.0 AND BSD-3-Clause OR MIT"
```

Comment:
Is there any scanner that could create such a license finding? I think ScanCode would just create separate findings for each license?
What is the purpose of using an SPDX expression here? It's not clear from the commit message why you want to illustrate this.

Reply:
The purpose is to test how the reporter behaves in case of that particular license.
Such license won't come from the scanner directly but it could come from a license finding curation. I didn't want to mess around with a curations file which is why I added the ID here.

Further I think it makes sense to treat curated and uncurated license findings in the same way, that is support SPDX expressions for both.

What do you think?

Reply:
This should be explained in the commit message.

Reply:
Updated the commit message.